eng/tools/generator/cmd/v2/release/releaseCmd.go

```diff
@@ -111,6 +122,14 @@ func (c *commandContext) execute(sdkRepoParam, specRepoParam string) error {
 		return err
 	}
 
+	if path.Ext(c.rpName) == ".json" {
+		return c.generate2(sdkRepo, specRepoParam, specCommitHash)
```

Comment:
Could you please use a more readable naming for the function?

Reply:
generateFromRequest